libs/server/Storage/Session/ObjectStore/SortedSetOps.cs

```diff
@@ -445,11 +445,11 @@ public unsafe GarnetStatus SortedSetRange<TObjectContext>(ArgSlice key, ArgSlice
             {
                 case SortedSetOrderOperation.ByScore:
                     sortedOperation = SortedSetOperation.ZRANGEBYSCORE;
-                    operation = Encoding.ASCII.GetBytes("BYSCORE");
+                    operation = "BYSCORE"u8.ToArray();
```

Comment:
Shouldn't these arrays be made `static readonly` as class members instead?

Reply:
That'd be improvement yes, I kept changes minimal but this one is easy enough, thanks! 👍